profile/v20200901/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - profile/v20200901/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - profile/v20200901/
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    IncludeRelease: true
+    ServiceDirectory: '../profile/v20200901'
```

Comment:
How frequently will new profiles be added, that is to say, do we need to think about a large amount of pipelines getting created that differ by profile date/version?

Reply:
I think there won't be too much new profile, maybe one per year or even longer interval, it is only for Azure Stack. Thus, I could bear with little manual work to let pipeline work.